plugins/scaffolder-actions/scaffolder-backend-module-http-request/src/actions/run/backstageRequest.ts

```diff
@@ -78,6 +79,12 @@ export function createHttpBackstageAction(options: {
             description: 'The body you would like to pass to your request',
             type: ['object', 'string'],
           },
+          logging: {
```

Comment:
Maybe we could name this something else? I'm not really sure what a more descriptive name could be though :thinking:

Reply:
I was thinking this also but didn't come up with a more fitting name

Reply:
maybe just `logRequestPath` ?

Reply:
Yeah that sounds good!